README.md

```diff
@@ -72,6 +76,11 @@ npm install
 npm run example:gif # generates both gif and video
 ```
 
+### Installing the tool as globally
+```sh
+npm install -g .
+```
```

Comment:
do you need that?
I mean "to checkout repo, and after that installing it globally"

You could install it globally by usual:
```
npm i site-recorder -g
```

or using it without installation by
```
npx site-recorder ...
```

Reply:
You right and I fix documentation. I think its better not to do the npx each time as its running all preRun activities on each run. So I update that install will include the global